stellarphot/photometry/source_detection.py

```diff
@@ -127,29 +104,56 @@ def compute_fwhm(
         except AttributeError:
             pass
 
-        cutout = Cutout2D(ccd.data, (x, y), 5 * fwhm_estimate)
-        if fit:
-            fit = _fit_2dgaussian(cutout.data)
-            fwhm_x.append(gaussian_sigma_to_fwhm * fit.x_stddev_1)
-            fwhm_y.append(gaussian_sigma_to_fwhm * fit.y_stddev_1)
-            # print('Still fitting!!')
+        # SKY SUBTRACT SHIT!!
```

Comment:
While one of the milder swears, doesn't seem to particularly be necessary, was this really that hard. :)

Reply:
You have no idea....but I'll take it out.